### PR TITLE
Added BMS Tutorial Notebook

### PR DESCRIPTION
**Pull request by TheLemonPig**
The full notebook is added under the name "Tutorial". I figured it would help to keep the simple use case created by Ben, for those less familiar with coding.

### CODE REVIEW

**Review by benwandrew — COMMENTED** [review]
awesome, thanks! my only two cents are the following:
1. shall we title this something more descriptive than "Tutorial"?
2. can we include one or two sentences at the very top of the notebook which give some context / lay out the objective(s) of the notebook? eg, "in this tutorial, we will walk through how to [abc], [xyz]... we begin by..."

**Comment by TheLemonPig** [comment]
> awesome, thanks! my only two cents are the following:
> 
> 1. shall we title this something more descriptive than "Tutorial"?
> 2. can we include one or two sentences at the very top of the notebook which give some context / lay out the objective(s) of the notebook? eg, "in this tutorial, we will walk through how to [abc], [xyz]... we begin by..."

Good points. I'll create some suggested changes and see what you think about them before committing

**Review by musslick — CHANGES_REQUESTED** [review]
Looks great! The only item is that I would change the title to "Automated Scientific Discovery with the Bayesian Machine Scientist".

File: mkdocs.yml [hunk 1]
```diff
@@ -12,3 +12,4 @@ nav:
 - Search Space: 'search-space.md'
 - Examples:
     - Basic Usage: 'Basic Usage.ipynb'
+    - Tutorial: 'Tutorial.ipynb'
```
Comment (TheLemonPig) [inline mkdocs.yml]:
```suggestion
    - Tutorial: 'Automated Scientific Discovery with the Bayesian Machine Scientist.ipynb'
```

**Review by benwandrew — APPROVED** [review]
terrific, thanks for the updates!

**Comment by TheLemonPig** [comment]
@benwandrew we should close this PR now instead of merging, right? Since this BMS Tutorial is now being used as the Equation Discovery Tutorial in ```autora```

**Comment by benwandrew** [comment]
> @benwandrew we should close this PR now instead of merging, right? Since this BMS Tutorial is now being used as the Equation Discovery Tutorial in `autora`

i think that's right; we should probably close this now...

**Pull request closed without merging**